CLIENT-SPECIFICATION.md

```diff
@@ -232,35 +230,36 @@ including the changes. NOTE: tagging of the commit with a new version tag (in
 the form `vX.Y`) should be done immediately AFTER merging the version bump, as
 the commit hash changes when merging with squash or rebase.
 -->
- - Unreleased
-
- - [v2.0, September 10th 2023](https://github.com/tldr-pages/tldr/blob/v2.0/CLIENT-SPECIFICATION.md) ([#10148](https://github.com/tldr-pages/tldr/pull/10148))
-   - Add recommendation to support `macos` alias for `osx` ([#7514](https://github.com/tldr-pages/tldr/pull/7514))
-   - Drop the special "all" platform from the `--list` flag ([#7561](https://github.com/tldr-pages/tldr/pull/7561))
-   - Drop the `master` branch from the assets link. ([#9668](https://github.com/tldr-pages/tldr/pull/9668))
-   - Require support for long options ([#9651](https://github.com/tldr-pages/tldr/pull/9651))
-   - Add recommendation to support caching individual translation archives ([#10148](https://github.com/tldr-pages/tldr/pull/10148))
-
- - [v1.5, March 17th 2021](https://github.com/tldr-pages/tldr/blob/v1.5/CLIENT-SPECIFICATION.md) ([#5428](https://github.com/tldr-pages/tldr/pull/5428))
-   - Add requirement for converting command names to lowercase before running the page resolution algorithm.
-   - Use HTTPS for archive links.
-
- - [v1.4, August 13th 2020](https://github.com/tldr-pages/tldr/blob/v1.4/CLIENT-SPECIFICATION.md) ([#4246](https://github.com/tldr-pages/tldr/pull/4246))
-   - Add requirement for CLI clients to use non-zero exit code on failing to find a page.
-
- - [v1.3, June 11th 2020](https://github.com/tldr-pages/tldr/blob/v1.3/CLIENT-SPECIFICATION.md) ([#4101](https://github.com/tldr-pages/tldr/pull/4101))
-   - Clarified fallback to English in the language resolution algorithm.
-   - Update the `LANG` and `LANGUAGE` environment variables to conform to the GNU spec.
-
- - [v1.2, July 3rd 2019](https://github.com/tldr-pages/tldr/blob/v1.2/CLIENT-SPECIFICATION.md) ([#3168](https://github.com/tldr-pages/tldr/pull/3168))
-   - Addition of a new `-L, --language` recommended command-line option.
-   - Rewording of the language section, also encouraging the use of configuration files for language.
-   - Shift from BCP-47 to POSIX style locale tags, with consequent **deprecation of previous versions of the spec**.
-   - Clearer clarification about the recommended caching functionality.
-   - Correction of the usage of the term "arguments" in the homonym section.
-
- - [v1.1, April 1st 2019](https://github.com/tldr-pages/tldr/blob/v1.1/CLIENT-SPECIFICATION.md) (deprecated) ([#2859](https://github.com/tldr-pages/tldr/pull/2859))
-   - Clarified platform section.
-
- - [v1.0, January 23rd 2019](https://github.com/tldr-pages/tldr/blob/v1.0/CLIENT-SPECIFICATION.md) (deprecated) ([#2706](https://github.com/tldr-pages/tldr/pull/2706))
-   - Initial release.
+
+- Unreleased
```

Comment:
Changelog will need updating before this is merged.

Reply:
Agreed, will update the changelog now and will merge this PR tomorrow (will prepare the release notes too).

Reply:
Done, updated the changelog. Now, I will work on the release notes and then the new release is GTG.